boards/IoT_6LoWPAN_gateway_v01/README.md

```diff
@@ -0,0 +1,44 @@
+# IoT_6LoWPAN_gateway_v01
+
+![Logo](figures/IoT_6LoWPAN_gateway_v01_mounted.jpg)
+
+
+**Board description**
+
+This project aims to add WSN (Wireless Sensor Network) connectivity to the DragonBoard 410C for applications that need low-cost, long-range and low data rate communication. The gateway should be able to operate in remote environments without direct 
+human intervention. 
+
+The IoT_6LoWPAN_gateway_v01 board features:
+- 6loWPAN Sub-1GHz Radio (CC1310).
+- 3G modem (Telit HE910-EUD). 
+- Circuitry for automatic switch between AC Adapter power supply and backup battery. 
+- Ultra-low power MCU to manage the board, monitor voltages, and works as an external watchdog with capability to reset the DragonBoard in case of error. 
+- Noise, short circuit and overvoltage protection
+- Shutdown and reset relay
+- Power on LED
+- A internal 3.3V regulator to power radio module and board circuitry.
+
+
+*Attention: Power supply or battery is expected to have a nominal voltage of 12V, but it supports 8V to 15V. The DragonBoard is powered from the SYS_DCIN pins and is attached by the Low Speed Expansion Connector.
+
+<p align="center">
+<img style=" display:center" src="figures/board_diagram.png" width="500" alt=""/>
```

Comment:
This has not yet been outlined in the contribute.md yet, but we should try to form some naming conventions.

Keeping everything in "lower case" is good, use "-" instead of "_" and be sure to keep all file names as simple as possible.

We will try to come up with some Guidelines as soon as possible.